docs/multi-cluster-ingress.md

```diff
@@ -27,55 +27,45 @@ We use [multi-cluster-ingress-nginx](https://github.com/karmada-io/multi-cluster
 
 ```shell
 # for HTTPS
-git clone https://github.com/karmada-io/multi-cluster-ingress-nginx.git
+$ git clone https://github.com/karmada-io/multi-cluster-ingress-nginx.git
 # for SSH
-git clone git@github.com:karmada-io/multi-cluster-ingress-nginx.git
+$ git clone git@github.com:karmada-io/multi-cluster-ingress-nginx.git
 ```
 
 #### Build and deploy ingress-nginx
 
 Using the existing `karmada-host` kind cluster to build and deploy the ingress controller.
 
 ```shell
-export KIND_CLUSTER_NAME=karmada-host
-kubectl config use-context karmada-host
-make dev-env
+# set the default cluster profile
+$ export KUBECONFIG=~/.kube/karmada.config
+# set the kind cluster name for ingress deploy
+$ export KIND_CLUSTER_NAME=karmada-host
+# the default context is switched to karmada-host
+$ kubectl config use-context karmada-host
+Switched to context "karmada-host".
+$ cd multi-cluster-ingress-nginx
+$ make dev-env
```

Comment:
The steps are more detailed, but the step comments are a little redundant. And is it necessary to add a $ symbol before the command, if so, it may be unfriendly for direct copying.

Reply:
Do you need me to remove all the $？